src/views/DandisetLandingView/DandisetOwners.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <v-card
+    v-if="publishDandiset"
+    height="100%"
+    class="px-3 py-1"
+    outlined
+  >
+    <v-row
+      no-gutters
+      class="my-1"
+    >
+      <v-card-subtitle>
+        Ownership
+      </v-card-subtitle>
+      <template v-if="!owners || !owners.length">
+        <v-row
+          align="center"
+          justify="center"
+          class="mx-1"
+        >
+          No owners
+        </v-row>
+      </template>
+      <template v-else>
+        <v-tooltip
+          top
+          :disabled="!manageOwnersDisabled"
+        >
+          <template #activator="{ on }">
+            <div v-on="on">
+              <v-row class="mx-1 px-1">
+                <v-autocomplete
+                  v-model="selection"
+                  :items="items"
+                  :disabled="manageOwnersDisabled"
+                  :loading="loadingUsers"
+                  :search-input.sync="search"
+                  hide-no-data
+                  clearable
+                  auto-select-first
+                  item-text="result"
+                  placeholder="enter email address"
+                  outlined
+                  flat
+                  return-object
+                  @update:search-input="throttledUpdate"
+                />
+              </v-row>
+            </div>
+          </template>
+          <template v-if="loggedIn">
+            You must be an owner to manage ownership.
+          </template>
+          <template v-else>
+            You must be logged in to manage ownership.
+          </template>
+        </v-tooltip>
+        <v-row class="mx-1">
+          <v-chip
+            v-for="(owner, i) in newOwners"
+            :key="i"
+            color="light-blue lighten-4"
+            text-color="light-blue darken-3"
+            class="font-weight-medium ma-1"
+            :close="userCanModifyDandiset"
+            @click:close="removeOwner(i)"
+          >
+            {{ owner.name || owner.username }}
+          </v-chip>
+        </v-row>
+      </template>
+    </v-row>
+  </v-card>
+</template>
+
+<script>
+import { publishRest, loggedIn, user } from '@/rest';
+import { mapState, mapMutations } from 'vuex';
+import _ from 'lodash';
+
+// Includes a field `result` on each user which is the value displayed in the UI
+const appendResult = (users) => users.map((u) => ({ ...u, result: (u.name) ? `${u.name} (${u.username})` : u.username }));
+
+export default {
+  name: 'DandisetOwners',
+  props: {
+    owners: {
+      type: Array,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      search: null,
+      loadingUsers: false,
+      selection: null,
+      newOwners: appendResult(this.owners),
+      items: [],
+      throttledUpdate: _.debounce(this.updateItems, 200),
+    };
+  },
+  asyncComputed: {
+    async userCanModifyDandiset() {
```

Comment:
In `DandisetLandingView` this same value is calculated using a `ref` and a `watchEffect`. If it's used in multiple components like this it's probably worth moving it to the store or going all in on the props thing and only calculating it once.

Reply:
Oh I realized after seeing that `asyncComputed` that I didn't merge the TS/Composition conversionI made to `DandisetOwners` into this branch, I did so in https://github.com/dandi/dandiarchive/pull/857/commits/1433effd00f0de2a5079679ad9a150c147cc0f8d

I moved `userCanModifyDandiset` into the store https://github.com/dandi/dandiarchive/pull/857/commits/00d82e4716a5313c81ea513049da456677803d57, it seems to fit in nicely as a getter.